1-js/04-object-basics/06-constructor-new/article.md

```diff
@@ -213,17 +212,17 @@ john = {
 */
 ```
 
-## Summary
+## Sumário
 
-- Constructor functions or, briefly, constructors, are regular functions, but there's a common agreement to name them with capital letter first.
-- Constructor functions should only be called using `new`. Such a call implies a creation of empty `this` at the start and returning the populated one at the end.
+- Funções construtoras ou, resumidamente, construtores, são funções regulares, mas existe um acordo comum para nomeá-las com a primeira letra em maiúsculo.
+- Funções construtoras deveriam ser chamadas apenas usando `new`. Tal chamada implica a criação de um `this` vazio no começo e retornando o completo ao final.
 
-We can use constructor functions to make multiple similar objects.
+Nós podemos usar funções construtoras para fazer diversos objetos iguais.
 
-JavaScript provides constructor functions for many built-in language objects: like `Date` for dates, `Set` for sets and others that we plan to study.
+JavaScript fornece funções construtoras para muitos objetos de linguagem internos: como `Date` para datas, `Set` para armazenamento e outros que nós planejamos estudar.
```

Comment:
- "...muitos objetos internos <s>de</s> da linguagem <s>internos</s>:"
- ", `Set` para <s>armazenamento</s> conjuntos e outros..."
('Set' in Algebra)